src/org/mozilla/javascript/AbstractEcmaObjectOperations.java

```diff
@@ -24,6 +24,32 @@
         SEALED
     }
 
+    /**
+     * Implementation of Abstract Object operation HasOwnProperty as defined by EcmaScript
+     *
+     * @param cx
+     * @param o
+     * @param pn
+     * @return boolean
+     * @see <a href="https://tc39.es/proposal-accessible-object-hasownproperty/"></a>
+     */
+    static boolean hasPropertyInObject(Context cx, Object o, Object pn) {
+        ScriptableObject obj = ScriptableObject.ensureScriptableObject(o);
+        boolean result;
+        if (pn instanceof Symbol) {
+            result = obj.has((Symbol) pn, obj);
```

Comment:
Don't you need `ensureSymbolScriptable`?
https://github.com/mozilla/rhino/blob/5a8d31100cf840d2d5746c25d10c0c7f52f8fad0/src/org/mozilla/javascript/NativeObject.java#L210

Reply:
> Don't you need `ensureSymbolScriptable`?
> 
> https://github.com/mozilla/rhino/blob/5a8d31100cf840d2d5746c25d10c0c7f52f8fad0/src/org/mozilla/javascript/NativeObject.java#L210

When I tested, it went well without the method, but there was a problem that I didn't predict, so I'll fix it in the next commit! Thank you for pointing out.